tests/webidl/post.js

```diff
@@ -287,5 +287,14 @@ if (isMemoryGrowthAllowed) {
 
 //
 
+const fs = require('fs')
+const data = fs.readFileSync('glue.cpp', 'utf8')
+if (data.includes("int uCharArrayTest(const unsigned char* arg, int size)")) {
```

Comment:
What would this text have been before this fix, btw?

Reply:
The current implementation would generate `int uCharArrayTest(const unsigned char arg, int size)` for the JS implementation

Reply:
Got it, thanks! Makes sense.